dmsnitch/resource_snitch.go

```diff
@@ -16,6 +17,7 @@ type Snitch struct {
 	Status   string   `json:"status,omitempty"`
 	Interval string   `json:"interval,omitempty"`
 	Type     string   `json:"alert_type,omitempty"`
+	Email    []string `json:"alert_email,omitempty"`
```

Comment:
```suggestion
	AlertEmails   []string `json:"alert_email,omitempty"`
```
Let's be specific what "email" is for